recipes/views.py

```diff
@@ -157,4 +157,4 @@ class CreateIngredientView(CreateAPIView):
 
 class ListIngredientView(ListAPIView):
     serializer_class = IngredientSerializer
-    queryset = Ingredient.objects.all()
\ No newline at end of file
+    queryset = Ingredient.objects.all()
```

Comment:
This should be `ListCreateAPIView`. Did you run the tests? I think, status code of `POST` request should be `405` for `ListAPIView`.